x/trigger/keeper/trigger.go

```diff
@@ -0,0 +1,101 @@
+package keeper
+
+import (
+	types "github.com/cosmos/cosmos-sdk/codec/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+
+	triggertypes "github.com/provenance-io/provenance/x/trigger/types"
```

Comment:
Usually, in the keepers, the "types" import is the `/types` of the module, and any other `/types` imports have aliases. Also, you shouldn't include an "alias" when its not different from the import.

```suggestion
	codectypes "github.com/cosmos/cosmos-sdk/codec/types"
	sdk "github.com/cosmos/cosmos-sdk/types"

	"github.com/provenance-io/provenance/x/trigger/types"
```

(uses below will need updating with this change).